tests/net/rpl/src/main.c

```diff
@@ -510,7 +467,7 @@ static bool populate_nbr_cache(void)
 
 	if (data_failure) {
 		data_failure = false;
-		return false;
+		zassert_true(0, NULL);
```

Comment:
can't you just zassert on data_failure directly?

Reply:
fixed